includes/wikia/nirvana/WikiaSkin.class.php

```diff
@@ -276,15 +276,17 @@ public function getStylesWithCombinedSASS(Array &$sassFiles) {
 		$sassFiles = $this->assetsManager->getSassFilePath($sassFiles);
 
 		// get a single URL to fetch all the required SASS files
-		$sassFilesUrl = $this->assetsManager->getSassesUrl($sassFiles);
+		$sassParams = SassUtil::getSassSettings();
+		$sassParams['crossorigin'] = 'anonymous';
```

Comment:
I give this approach a second thought and...  Instead of using the URL parameter and passing it through several layers of code we can use `config.php` file and define a flag for AM group to set `crossorigin` header to a given value when serving a specific group.

* less code changes
* we have better control over which assets can be served with `crossorigin` header